R/create_map.R

```diff
@@ -13,6 +13,7 @@
 #' @param left_lon left longitude
 #' @param top_lat top latitude
 #' @param bottom_lat bottom latitude
+#' @importFrom rlang .data
```

Comment:
I have no experience with this idiom. It feels weird to import from `rlang`... well, I just assume this works, because the tests say so :+1: